GW2EIEvtcParser/EIData/DamageModifiers/CommonDamageModifiers/EncounterDamageModifiers.cs

```diff
@@ -67,6 +67,7 @@ internal static class EncounterDamageModifiers
                     Segment segment = target.GetBuffStatus(log, IonShield, ahde.Time);
                     return segment.Value >= 20;
                 }),
+            new BuffOnFoeDamageModifier(IcyBarrier, "Icy Barrier", "-10% per stack", DamageSource.All, -10.0, DamageType.StrikeAndCondition, DamageType.All, Source.FightSpecific, ByStack, BuffImages.ShieldOfIce, DamageModifierMode.PvE),
```

Comment:
This combines with vulnerability and it is also possible for it to reach 100% reduction. Check out how that is managed on PowerOfTheVoid, DagdaDemonicAura, PillarPandemonium